lib/tokenize.cpp

```diff
@@ -1019,20 +1019,23 @@ namespace {
     };
 }
 
-bool Tokenizer::isFunctionPointer(const Token* tok) {
-    return Token::Match(tok, "%name% ) (");
-}
-
 void Tokenizer::simplifyTypedef()
 {
     // Simplify global typedefs that are not redefined with the fast 1-pass simplification.
     // Then use the slower old typedef simplification.
-    std::map<std::string, int> numberOfTypedefs;
+    std::map<std::string, std::set<std::string>> numberOfTypedefs;
     for (Token* tok = list.front(); tok; tok = tok->next()) {
         if (tok->str() == "typedef") {
             TypedefSimplifier ts(tok);
+            if (ts.name().empty())
```

Comment:
imho it's a bit ugly to construct a string here. I prefer
```suggestion
            if (ts.nameToken())
```